app/src/main/java/com/cornellappdev/android/eatery/model/CafeModel.java

```diff
@@ -134,6 +142,51 @@ public void setHours(LocalDate date, List<Interval> hours) {
 		this.mHours.put(date, sortedHours);
 	}
 
+	public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eatery cafe){
+		super.parseEatery(context, hardcoded, cafe);
+		List<String> cafeItems = new ArrayList<>();
+
+		DateTimeFormatter timeFormatter = new DateTimeFormatterBuilder()
+				.parseCaseInsensitive()
+				.appendPattern("h:mma")
+				.toFormatter();
+		DateTimeFormatter dateFormatter = new DateTimeFormatterBuilder()
+				.parseCaseInsensitive()
+				.appendPattern("yyyy-MM-dd")
+				.toFormatter();
+
+		for(AllEateriesQuery.OperatingHour operatingHour : cafe.operatingHours()){
+			List<LocalDate> localDates = new ArrayList<>();
+			LocalDate localDate = LocalDate.parse(operatingHour.date(),dateFormatter);
+			localDates.add(localDate);
+			for(AllEateriesQuery.Event event : operatingHour.events()){
+				List<Interval> dailyHours = new ArrayList<>();
+				for(AllEateriesQuery.Menu menu : event.menu()){
+					for(AllEateriesQuery.Item item : menu.items()){
```

Comment:
space after `)`, so `) {`, can you set up a linter so we can avoid linting fixes in the future?